.gitignore

```diff
@@ -8,3 +8,7 @@ test_products
 /test_wallets/
 /build
 /browser_build/
+unbuild.sh
```

Comment:
As you know, generating a Monero build requires a "clean slate" to finish correctly; in other words, you have to delete all the folders generated by any previous build attempts. Because I had to rebuild numerous times before finally finding success, I created a shell script to automate the cleanup process between builds. Because this script was for my own personal use and not something that needed to be included with any published release of monero-javascript, I added it to the .gitignore.